reference/fleet/alerting-rule-templates.md

```diff
@@ -0,0 +1,49 @@
+---
+applies_to:
+  stack: ga 9.3
+  serverless: ga
+products:
+  - id: fleet
+  - id: elastic-agent
+navigation_title: Alerting rule templates
+---
+
+# Alerting Rule Templates [alerting-rule-templates]
+
+Alerting rule templates are out-of-the-box, preconfigured rule definitions maintained by Elastic integration authors. They help you start monitoring in minutes—no queries to write, no thresholds to figure out—by providing curated {{esql}} queries, sensible defaults, and recommended thresholds tailored to each integration. Templates are available from an integration’s Assets and open a prefilled rule creation form you can adjust and enable.
+
+## Prerequisites
+	
+- Install or upgrade to the latest version of the integration that includes alerting rule templates.
+- Ensure the relevant data stream is enabled and ingesting data for the template you plan to use.
+- {{stack}} 9.2.1 or later.
+- Appropriate {{kib}} role privileges to create and manage rules in the current space.
+
+## How to use the Alerting Rule Templates
+
+Alerting rule templates come with recommended, pre-populated values. To use them:
```

Comment:
these instructions will become out of date v quickly (by 9.3). let's open a ticket to remind us to update them with the new UI flow for the next release.